packages/tenant-process/src/services/tenantService.ts

```diff
@@ -669,6 +652,43 @@ async function assignCertifiedAttribute({
   return updatedTenant;
 }
 
+function buildVerifiedBy(
+  verifiedTenantAttribute: VerifiedTenantAttribute,
+  organizationId: TenantId,
+  tenantAttributeSeed: ApiVerifiedTenantAttributeSeed
+): TenantVerifier[] {
+  const hasPreviouslyVerified = verifiedTenantAttribute.verifiedBy.find(
+    (i) => i.id === organizationId
+  );
+  return hasPreviouslyVerified
+    ? verifiedTenantAttribute.verifiedBy.map((verification) =>
+        verification.id === organizationId
+          ? {
+              id: organizationId,
+              verificationDate: new Date(),
+              expirationDate: tenantAttributeSeed.expirationDate
+                ? new Date(tenantAttributeSeed.expirationDate)
+                : undefined,
+              extensionDate: tenantAttributeSeed.expirationDate
+                ? new Date(tenantAttributeSeed.expirationDate)
+                : undefined,
```

Comment:
```suggestion
              extensionDate: undefined,
```
This field has to be set to `undefined`